src/daemons/rhsmcertd.c

```diff
@@ -433,7 +422,7 @@ auto_register(gpointer data)
 }
 
 static gboolean
-cert_check (gboolean heal)
+cert_check ()
```

Comment:
this is C, so you need to specify `void` in case a function takes no parameters; actually, since this function is called as timeout (see `g_timeout_add()`), then the right parameter is `gpointer data`; to avoid warnings about the unused parameter, use `G_GNUC_UNUSED`:
```suggestion
cert_check (G_GNUC_UNUSED gpointer data)
```